### DISCUSSION TITLE
Moon 2 Mars Fest Schedule

### DISCUSSION THREAD

**Discussion by ISS-Mimic** [ISS-Mimic/Mimic#372] [General]
### Moon 2 Mars Shift Scheduling

 <html>
<body>

| **Mimicker** |  **Sat AM 3/11** |  **Sat PM 3/11** |  **Sun AM 3/12** |  **Sun PM 3/12** |  **Mon AM 3/13** |  **Mon PM 3/13** |  **Tue AM 3/14** |  **Tue PM 3/14** |  **Wed AM 3/15** |  **Wed PM 3/15** |  **Thu AM 3/16** |  **Thu PM 3/16** |  **Fri AM 3/17** |  **Fri PM 3/17** |  **Sat AM 3/18** |  **Sat PM 3/18** |  **Sun AM 3/19** |  **Sun PM 3/19** |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
Sam | ✔ | ✔ | ❌ | ❌ | ❌ | ❌ | ❌| ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ✔ | ✔ | ✔ |
Chen | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔| ✔ | ❔ | ❔ | ❔ | ❔ | ❌ | ❔ | ✔ | ✔ | ✔ | ✔ |
Aya | ✔ | ✔ | ❌ | ✔ | ✔ | ✔ | ✔| ❌ | ❌ | ❌ | ❔ | ❔ | ❌ | ✔ | ❌ | ❌ | ✔ | ✔|
Susan | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔| ✔ | ✔ |✔ | ❌ | ❌ | ❌ |❌ | 🔵 | 🔵 | ❌ | ❌ |
Doug | ✔ | ✔| ✔ | ✔ | ➖ | ➖ | ➖| ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ |
Craig | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖| ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ |
Bryan | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
Tristan | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖| ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ | ➖ |
Dallas | ❔ | ❔ | ❔ | ❔ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❔ | ❔ |

✔ = Available for shift
❌ = Not available for shift
🔵 = Partial support
➖ = Not filled out
❔ = Not sure yet

Link to other icons/emojis = https://gist.github.com/rxaviers/7360908 

</body>
</html>

**Comment by GodspeedDiscvry** [comment]
I think I will be here for both weekends.  But if I m headed out to Ok... I don't want to sign up for every shift because I would need to pack and stuff.  As for the weekdays, I can probably pick one day that I am needed and take that day off.

**Comment by GodspeedDiscvry** [comment]
Wow... looking at my schedule now - they put this in the worst possible space.

**Comment by GodspeedDiscvry** [comment]
I may need to bug out for an hour on Saturday/Sunday/or both.  And I have not really decided on Monday and/or Tuesday yet.  I guess I will support however needed but not sure I can do both full days.
